src/orange/surf/ConeAligned.hh

```diff
@@ -126,11 +129,11 @@ CELER_CONSTEXPR_FUNCTION SurfaceType ConeAligned<T>::surface_type()
 /*!
  * Construct from origin and tangent of the angle of its opening.
  *
- * Given the triangular cross section of one octant of a finite cone (i.e. a
- * right triangle), the tangent is the slope of its hypotenuse (height / base).
+ * Given a finite cone, the tangent is ratio of the radius at its base to the
+ * its height.
```

Comment:
The english here is broken.
(I can't find how to suggest changes myself, that button does not show up for me)

Reply:
Misplaced `the` 😄 I'll fix it.